README/ReleaseNotes/v628/index.md

```diff
@@ -182,12 +182,42 @@ RNTuple is still experimental and is scheduled to become production grade in 202
 ### New features
 
 - Add [`GraphAsymmErrors`](https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html#acea30792eef607489d498bf6547a00a6) action that fills a TGraphAsymmErrors object.
+- Introduce [`RDatasetSpec`](https://root.cern/doc/master/classROOT_1_1RDF_1_1Experimental_1_1RDatasetSpec.html) as an
+experimental class to specify the input dataset to an RDataFrame.
+- Introduce metadata handling in the specification of the input dataset. An RDataFrame can also be created directly from
+a specification (including metadata) in JSON format.
```

Comment:
The wording about metadata handling is a bit abstract, I'm afraid users won't get it: maybe "`DefinePerSample` can now access arbitrary metadata that is associated to a sample via `RDatasetSpec`. See also ..." or similar

Creating an RDataFrame directly from a JSON file with a dataset specification probably deserves its own line (with link to the function)